app/templates/components/donations/create-donation.hbs

```diff
@@ -1,11 +1,10 @@
+<h3>Support this project each month</h3>
 <div class="amounts">
 {{#each presetAmounts as |presetAmount|}}
   {{donations/donation-amount-button
-    customAmount=customAmount
     presetAmount=presetAmount
-    selectedAmount=amount
-    setCustomAmount=(action (mut customAmount))
-    setAmount=(action (mut amount))
+    click=(action (mut amount) presetAmount)
+    selected=(and (eq presetAmount (float amount)) (not customAmount))
```

Comment:
A lot of data was being set down for the sake of computing the flag. I figured, makes more sense to compute the flag where the data is and send that flag down instead.

The part that is weird to me is the `(not customAmount)`. This statement here basically says, "if amount is equal to one of the 4 button amounts, render that button as selected, unless that amount is written in the custom amount box.

It would make sense to highlight the appropriate button in that case as well, but there is a test that explicitly ensures that behavior is as defined here, so I did not touch it.